iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/PublicAccessCheck.java

```diff
@@ -71,8 +80,25 @@ protected void registerResourceConsumer() {
       resource -> resource.attribute("visibility")
         .reportIf(equalTo("public"), MESSAGE)
         .reportIfAbsent(OMITTING_DNS));
```

Comment:
I thought you were not `String.format`-ing the right message here, but then I looked at the implementation of `ifAbsent(...)`